SortingAlgorithm/SortingAlgorithm/SortingAlgorithm.h

```diff
@@ -0,0 +1,44 @@
+// SortingAlgorithm.h
+
+#pragma once
+
+using namespace System;
+
+namespace SortingAlgorithm {
+	public ref class Sort
+	{
+	public:
+		virtual void sort(int arr[], int size);
+	};
+	public ref class MergeSort: public Sort
```

Comment:
Plese write separate header files for each of them.

Reply:
I've created this header file as  a common header file for all the sorting algorithm. Each of the algorithm is implemented into their corresponding cpp file. So, is it necessary to add header file for each of the sorting algorithm?
When people include this single header file, they can use any one of the 4 sorting algorithms implemented in the library project.